src/script/components/calling/CallParticipantsListItem/CallParticipantsListItem.tsx

```diff
@@ -69,41 +71,65 @@ export const CallParticipantsListItem = ({
     });
   };
 
+  const reactiveProps =
+    isAudioEstablished && showContextMenu
+      ? {
+          role: 'button',
+          tabIndex: TabIndex.FOCUSABLE,
+          onContextMenu,
+          onClick: onContextMenu,
+          onKeyDown: handleContextKeyDown,
+        }
+      : undefined;
+
   return (
     <div
-      tabIndex={TabIndex.FOCUSABLE}
-      role="button"
-      onContextMenu={onContextMenu}
-      onClick={onContextMenu}
-      onKeyDown={handleContextKeyDown}
+      {...reactiveProps}
       data-uie-name="item-user"
       data-uie-value={userName}
-      aria-label={t('accessibility.openConversation', userName)}
       css={callParticipantListWrapper(isLast)}
     >
       <div css={listItem(true)}>
-        <Avatar avatarSize={AVATAR_SIZE.SMALL} participant={user} aria-hidden="true" css={{margin: '0 10px'}} />
+        <Avatar
+          avatarSize={AVATAR_SIZE.SMALL}
+          participant={user}
+          aria-hidden="true"
+          css={{margin: '0 10px', opacity: isAudioEstablished ? '1' : '0.5'}}
+        />
 
         <CallParticipantItemContent
+          isAudioEstablished={isAudioEstablished}
           name={userName}
           selfInTeam={selfInTeam}
           availability={availability}
           isSelf={isSelf}
-          {...(showDropdown && {
-            onDropdownClick: event => onContextMenu?.(event as unknown as React.MouseEvent<HTMLDivElement>),
-          })}
-        />
-
-        <UserStatusBadges
-          config={{
-            guest: isDirectGuest,
-            federated: isFederated,
-            external: isExternal,
-            verified: isSelfVerified && isVerified,
-          }}
+          showContextMenu={showContextMenu}
+          onDropdownClick={event => onContextMenu?.(event as unknown as React.MouseEvent<HTMLDivElement>)}
         />
 
-        <CallParticipantStatusIcons callParticipant={callParticipant} />
+        {isAudioEstablished ? (
+          <>
+            <UserStatusBadges
+              config={{
+                guest: isDirectGuest,
+                federated: isFederated,
+                external: isExternal,
+                verified: isSelfVerified && isVerified,
+              }}
+            />
+            <CallParticipantStatusIcons callParticipant={callParticipant} />
+          </>
+        ) : (
+          <span
+            css={{
+              color: 'var(--red-500)',
```

Comment:
```suggestion
              color: 'var(--danger-color)',
```
the same as red-500 but a bit more descriptive, your call